pkg/catalogv2/git/content.go

```diff
@@ -13,16 +13,15 @@ import (
 	"helm.sh/helm/v3/pkg/repo"
 )
 
+// Icon will return the icon for a chartName version in a local repository by getting the relative path
 func Icon(namespace, name, gitURL string, chartVersion *repo.ChartVersion) (io.ReadCloser, string, error) {
 	if len(chartVersion.Icon) == 0 {
 		return nil, "", fmt.Errorf("failed to find chartName %s version %s: %w", chartVersion.Name, chartVersion.Version, validation.NotFound)
 	}
 
-	dir := gitDir(namespace, name, gitURL)
+	dir := RepoDir(namespace, name, gitURL)
 	icon := chartVersion.Icon
-	if strings.HasPrefix(icon, "file://") && len(chartVersion.URLs[0]) > 0 {
```

Comment:
The following use case is not working. 

1. Add https://rohitsakala.github.io/charts/ as ClusterRepo. (This points to the dev-v2.8 branch)
2. Check the Epinio Icon. It is not loading even though the epinio icon is present in https://rohitsakala.github.io/charts/assets/logos/epinio.svg